### PR TITLE
network faqs added

### PR DESCRIPTION
**Pull request by pmoncada**

### CODE REVIEW

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+**How do I fund my wallet?**
+Click on the top left purple button, in the dropdown select “fund” and follow the instructions. If you need support with this step you can create a ticket in our Discord here or email info@moondao.com.
+
+**How is data handled on the website?**
+We are an open source community, and data is stored onchain. This means it is written to the blockchain, please be aware that all data is public. You can request for your data to be removed from our front-end website, and then your profile will no longer show up on https://moondao.com. To delete your profile, navigate to your profile page and click the “edit” button and then "delete data."
+
+
+
+
+Teams FAQs:
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md]: Should be an H2 with ## like above

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Teams are companies, nonprofits, communities, or internal groups that are coordinating together using our tools to accelerate a permanent human presence on the Moon. MoonDAO teams interface directly with the MoonDAO community, allowing them access to fund, hire, and sell directly to the MoonDAO community.
+
+**How do I create a Team?**
+Currently you must apply to create a Team. Go to https://moondao.com/join and click “Create a Team” to fill out the application form. You should hear back within a few days.
+
+**What is the difference between a Manager and a Member role?**
+Managers can edit the team page directly, Members are displayed as part of the team and can access the teams portal but cannot edit the public team page.
+
+**How do I add team members?**
+On your team page, go to the members section and click “
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md]: Incomplete section

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+**Can I configure my organization differently?**
+Yes! If you can dream it you can build it with hats protocol. Our frontend only recognizes Manager and Member hats, but beyond that you can structure things however you’d like. If you’d like us to support your particular use case let us know.
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md:65]: Might not need to mention Hats Protocol here, but should be capitalized as proper noun to Hats Protocol

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+On your Team page click on “Treasury,” then on the Gnosis interface go to “Settings” and “Manage Safe Account Signers.” Don’t forget to update the number of confirmations needed to sign a transaction.
+
+**How do I extend my subscription?**
+At the top of the page click “Extend Plan” and enter the number of years you’d like to extend.
+
+**What happens if my subscription expires?**
+You will lose access to your portal and the features of the team. The assets in your wallet are still yours and can always be used regardless of your subscription.
+
+**How do I edit the bio or other page details?**
+Click on the pencil icon in the top right.
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md:77]: Specify going to Team profile page and then click?

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+**How do I add items to the marketplace?**
+Go to your team page and click “Create Listing”
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md:80]: Missing period

File: MoonDAO/docs/Network/NetworkFAQ.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+Nope! There are no middle-men. Sell directly to others.
+
+**How do I post on the job board?**
+On your team page click “Add a Job” to create a new posting.
+
+**How will I get notified if someone applies?**
+You can add an email or other form of notification for job postings, or link an existing form or job application website.
+
+**How do I create a proposal for funding?**
+On the team page click “Fund” and it will take you to our proposal system. This allows you to create a project with MoonDAO to fund an initiative. You can read more about projects <here>.
```
Comment (ryand2d) [inline MoonDAO/docs/Network/NetworkFAQ.md]: Missing link for < here >

File: MoonDAO/docs/Network/NetworkFAQ.md (hunk 1 above)
Comment (pmoncada) [reply MoonDAO/docs/Network/NetworkFAQ.md]: Thanks!

**Review by ryand2d — APPROVED** [review]

**Pull request merged**